app/src/main/java/Script/BookInfo.java

```diff
@@ -1,22 +1,27 @@
 package Script;
 
+import org.json.JSONArray;
+import org.json.JSONObject;
+
 import java.io.Serializable;
 
-public class BookInfo implements Serializable {
-	private String _projectName;
-	public String Name;
-	public String Abbr;
-	public int ChapterCount;
-	public int BookNumber;
+public class BookInfo implements Serializable
+{
+	private String projectName;
```

Comment:
This one is a tough call. I personally really like an obvious distinction between class member and local variables, and a leading underscore is the current convention in SIL code (mostly C#). But I see that Google at least does not do this for Java (https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s5.1-identifier-names). So I suppose you are right.